Core/Src/main.c

```diff
@@ -87,8 +89,21 @@ int main(void)
   /* Initialize all configured peripherals */
   MX_GPIO_Init();
   MX_USART2_UART_Init();
+  MX_SPI1_Init();
+  MX_TIM1_Init();
   /* USER CODE BEGIN 2 */
 
+  // Start Timer
+  HAL_TIM_PWM_Start(&htim1, TIM_CHANNEL_1);
+
+  uint8_t RxBuffer[3];
+  uint8_t TxBuffer[3] = {0b00000001, 0b10000000, 0b00000000};
```

Comment:
make TxBuffer a constant variable, and place it into the "USER CODE PV" section.